3D-microservices-observability/node-js-app/beachshirt/controllers/deliveryController.js

```diff
@@ -0,0 +1,47 @@
+/**
+* Driver for delivery service which is responsible to deliver the shirts of a given style.
+* @author Yogesh Prasad Kurmi (ykurmi@vmware.com)
+*/
+const uuidv1 = require('uuid/v1');
+
+module.exports = (app, log) => {
+
+    app.post('/dispatch/:orderNum/', (req, res) => {
+        let orderNum = req.params.orderNum;
+        let shirts = JSON.parse(req.body.shirts);
+        if (Math.floor(Math.random() * 5) == 0) {
+            let msg = "Failed to order shirts!";
```

Comment:
the error generation code could be extracted for reuse.  Also, the message is misleading, since this isn't the order service